From 7cefc6cc27333d098a9d112ef962f01b826edbc6 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Fri, 5 Jul 2013 12:15:26 +0900
Subject: [PATCH] Fix MATCH as FreeBSD

When only MATCH is used, FreeBSD matches kFreeBSD.
We need ^ in front of "FreeBSD".

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
---
 Dbus/src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Dbus/src/CMakeLists.txt b/Dbus/src/CMakeLists.txt
index 10d5232..4909067 100644
--- a/Dbus/src/CMakeLists.txt
+++ b/Dbus/src/CMakeLists.txt
@@ -29,7 +29,7 @@ add_definitions (-DMY_APPLET_DOCK_VERSION="${dock_version}")
 add_definitions (-DMY_APPLET_ICON_FILE="icon.svg")
 add_definitions (-DCD_PLUGINS_DIR="${pluginsdir}")
 
-IF (${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
+IF (${CMAKE_SYSTEM_NAME} MATCHES ^"FreeBSD")
 	set(kvm "kvm")
 endif()
 
-- 
2.30.0.rc2

